test/client/actions/admin_contactlist.test.js

```diff
@@ -42,6 +42,33 @@ describe('Add Campaign actions', () => {
     });
   });
 
+  describe('setCurrentContactList', () => {
+    const setCurrentContactListResult = setCurrentContactList(contactListFixture);
```

Comment:
Might be more explicit to call this "expectedAction" than "setCurrentContactListResult"